doc/guides/coverage.rst

```diff
@@ -127,27 +127,23 @@ You may postprocess these with your preferred tools. For example:
 Sanitycheck coverage reports
 ============================
 
-When targeting boards based on the POSIX architecture,
 Zephyr's :ref:`sanitycheck script <sanitycheck_script>` can automatically
 generate a coverage report from the tests which were executed.
-You just need to invoke it with the ``-C`` command line option.
+You just need to invoke it with the ``--coverage`` command line option.
 
-For example you may run::
+For supported emulator targets it's enough to just specify ``--coverage``::
```

Comment:
@andrewboie Today, this is a subsection of "Coverage reports using the POSIX architecture" (line 95).
Would it be clearer if changed in either of these ways:
a) Creating a new subsection for sanitycheck under "Test coverage reports in embedded devices or QEMU" above, and separate into it the QEMU description and leave this one for native_posix?
or
b) Moving this section in the hierarchy level 1 level up, so it is not nested inside the POSIX arch part?

Reply:
Since you left a -1 on this patch, please *clearly* state what your acceptance criteria are.

Reply:
I guess the easiest would be b) above.
So, change the bunch of "=" in line 128 for a bunch of "*" :-p

Reply:
OK, will do.